README.md

```diff
@@ -14,8 +14,23 @@ Also, we will aim at building utility that can enhance faster development of upc
 
 NB: When installing make sure you have admin priviledges otherwise, you will have to use `sudo npm i -g kemboijs-cli`
 
-Also, after make sure to edit the database url in file `src/models/sequelizeinstance.js`.
-You can now call or edit the database models. The created model is User, under `src/models/user.js`
+## Edit Database details
+Navigate to `src/models/sequelizeinstance.js`, edit the database name, username or password. 
+
+You can also add these credentials to the `.env` file.
+```
+  export DATABASE_URL = "postgres://user:pass@example.com:5432/dbname"
```

Comment:
You can add javascript for javascript codelines
```javascript
  export DATABASE_URL = "postgres://user:pass@example.com:5432/dbname"
```

Reply:
This is implemented.